src/scss/components/clients/addClient/addClientModal.scss

```diff
@@ -61,25 +61,51 @@
     .add-client-input-error {
       color: red !important;
       border: red 1px solid !important;
+      -webkit-box-shadow: -1px 1px 24px -2px rgb(203, 50, 50) !important;
+      -moz-box-shadow: -1px 1px 24px -2px rgb(203, 50, 50) !important;
+      box-shadow: -1px 1px 24px -2px rgb(203, 50, 50) !important;
     }
     .intermediate-block {
       margin-top: 12vh;
     }
+
     .group {
       width: 100%;
       height: 70%;
       padding-top: 20px;
+      margin-bottom: 15px;
+
+      input, textarea{
+        width: 100%;
+        font-size: 12px;
+        border-radius: 4px;
+        outline: none;
+        transition: 0.5s all ease-in-out;
+        border: 1px solid #20B2A9 !important;
```

Comment:
Using **!important** is permitted. Try another way for example specificity. (https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)